cmd/server/assets/codes/issue.html

```diff
@@ -35,7 +35,7 @@
       </div>
     {{end}}
 
-    <h1>Create verification code</h1>
+    <h1>{{t $.locale "create-verification-code"}}</h1>
```

Comment:
I'm not in love with having to pass in the locale, but template functions aren't mutable after creation.